typescript/src/api-doc-types/MemberDoc.ts

```diff
@@ -15,10 +14,12 @@ import { ModuleDoc } from './ModuleDoc';
  */
 export abstract class MemberDoc implements ApiDoc {
```

Comment:
Should this now extend `BaseApiDoc`?

Reply:
It should be possible, but since mostly everything is handled differently for `MemberDoc` TypeScript declarations, it won't make sense to extend from the `BaseApiDoc`.

e.g. name is computed differently, there is never an alias symbol and more. In my opinion that's something we could try to improve in a separate PR (_avoid some duplicate code parts_)